packages/theming/apple-theme/src/appleTheme.macos.ts

```diff
@@ -46,6 +46,76 @@ export const appleComponents = {
       variant: 'bodyStandard',
     },
   },
+  RadioButton: {
+    tokens: {
+      borderColor: 'rgba(0, 0, 0, 0.15)',
+      backgroundColor: 'white',
+    },
+    root: {
+      accessible: false,
+      focusable: false,
+      style: {
+        alignItems: 'flex-start',
+      },
+    },
+    button: {
+      style: {
+        backgroundColor: 'rgba(255, 255, 255, 1)',
```

Comment:
This isn't just 'white' ?

Reply:
Yes... I should probably create a token for that too